csharp/src/Google.Protobuf/JsonTokenizer.cs

```diff
@@ -177,6 +177,12 @@ private sealed class JsonTextTokenizer : JsonTokenizer
             // The set of states in which a value is valid next token.
             private static readonly State ValueStates = State.ArrayStart | State.ArrayAfterComma | State.ObjectAfterColon | State.StartOfDocument;
 
+            private const int MAX_BUILDER_SIZE = 360;
+            private const int DEFAULT_CAPACITY = 16;
+
+            [ThreadStatic]
+            private static StringBuilder? t_cachedInstance;
```

Comment:
Please rename to just `cachedInstance` - we don't have any `t_` prefixes anywhere as far as I'm aware.

Reply:
Or better, `cachedStringBuilder` (unless we introduce a new class)